### PR TITLE
Image processing algorithms added

### PR DESCRIPTION
**Pull request by binish784**
gray scale image enhancement using contrast stretching method :  histogram equalization added

### CODE REVIEW

**Comment by QuantumNovice** [comment]
Change `image_processing/Histogram Equalization` to `digital_image_processing/Histogram Equalization`

File: image_processing/Histogram Equalization/Contrast Stretch.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import cv2 
+import matplotlib.pyplot as plt
+import numpy as np
+
+last_list=[]
+rem=0
+L=256
+sk=0
+
+img=cv2.imread('inputImage.jpg',0)
```
Comment (QuantumNovice) [inline image_processing/Histogram Equalization/Contrast Stretch.py]: Change this to `img=cv2.imread('image_data/inputImage.jpg',0)`
Place images in `digital_image_processing/image_data`

File: image_processing/Histogram Equalization/Contrast Stretch.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+for i in range(len(x)):
+    prk=x[i]/k
+    sk=prk+sk
+    last=(L-1)*sk        
+    if rem!=0:
+        rem=int(last % last)
+    if rem >=0.5:
+        last=int(last)+1
+    else:
+        last=int(last)
```
Comment (cclauss) [inline image_processing/Histogram Equalization/Contrast Stretch.py]: 4 lines --> 1: __last = int(last + 1 if rem >=0.5 else last)__

File: image_processing/Histogram Equalization/Contrast Stretch.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+L=256
+sk=0
+
+img=cv2.imread('inputImage.jpg',0)
+
+x,y,z=plt.hist(img.ravel(),256,[0,256],label='x')
+k=np.sum(x)
+for i in range(len(x)):
+    prk=x[i]/k
+    sk=prk+sk
```
Comment (cclauss) [inline image_processing/Histogram Equalization/Contrast Stretch.py]: __sk += prk__  # These short variable names are old-school.

**Comment by cclauss** [comment]
==================================== ERRORS ==========================

 ERROR collecting digital_image_processing/Histogram Equalization/Contrast Stretch.py 
digital_image_processing/Histogram Equalization/Contrast Stretch.py:17: in <module>
    x,y,z=plt.hist(img.ravel(),256,[0,256],label='x')
E   AttributeError: 'NoneType' object has no attribute 'ravel'

________________ ERROR collecting scripts/validate_filenames.py ________________

scripts/validate_filenames.py:28: in <module>
    sys.exit(bad_files)
E   SystemExit: 2

------------------------------- Captured stdout --------------------------------

1 files contain uppercase characters:
digital_image_processing/Histogram Equalization/Contrast Stretch.py
1 files contain space characters:
digital_image_processing/Histogram Equalization/Contrast Stretch.py

**Pull request merged**